Tests/CreateAPITests/Snapshots.swift

```diff
@@ -5,26 +5,27 @@
 import Foundation
 import XCTest
 
-// TODO: Find it automatically
-var projectPath = ("~/Developer/CreateAPI/" as NSString).expandingTildeInPath
-
 func compare(expected: String, actual: String, file: StaticString = #file, line: UInt = #line) throws {
     let env = ProcessInfo.processInfo.environment
     let expectedURL = Bundle.module.resourceURL!
         .appendingPathComponent("Expected")
         .appendingPathComponent(expected)
     let actualURL = URL(fileURLWithPath: actual)
+    
     if env["GENERATE_SNAPSHOTS"] == "true" || !FileManager.default.fileExists(atPath: expectedURL.path) {
-        let projectPath = (projectPath as NSString).expandingTildeInPath
-        let destinationURL = URL(fileURLWithPath: projectPath + "/Tests/CreateAPITests/Expected/\(actualURL.lastPathComponent)")
+        let testsFolderPath = #filePath
+            .split(separator: "/")
+            .dropLast()
+            .joined(separator: "/")
```

Comment:
If this functional indentation style is not preferred, happy to inline.

Reply:
> If this functional indentation style is not preferred, happy to inline.

I like it 🙂 

> Speaking of style, should we make a lint process for us?

Great idea 

- #64